test/extended/image_ecosystem/mariadb_ephemeral.go

```diff
@@ -33,8 +33,8 @@ var _ = g.Describe("[image_ecosystem][mariadb][Slow] openshift mariadb image", f
 			g.It(fmt.Sprintf("should instantiate the template"), func() {
 				exutil.WaitForOpenShiftNamespaceImageStreams(oc)
 
-				g.By(fmt.Sprintf("calling oc process -f %q", templatePath))
-				configFile, err := oc.Run("process").Args("-f", templatePath).OutputToFile("config.json")
+				g.By(fmt.Sprintf("calling oc process %q", templatePath))
+				configFile, err := oc.AsAdmin().Run("process").Args(templatePath, "-n", "openshift").OutputToFile("config.json")
```

Comment:
why did this need to switch to AsAdmin?

Reply:
see https://github.com/openshift/origin/pull/24391#issuecomment-574193003 ... I started getting those permission errors consistently when the `oc process` calls tried to access templates in the openshift namespace instead of access a local file like was done previously.

Conversely, `oc new-app` did not get those errors (you'll see uses of that in this file that did not have to change).

The help for `oc process` does mention you can use `oc process openshift//foo` to access templates in the openshift namespace.  

But admittedly, I went for the path of least resistance at this point, given the delays in trying to get this in between CI infra flakes and 2nd and 3rd level discoveries of accessing upstream templates.

So we either:
a) go with this short cut
b) try `oc process openshift//` and see what happens
c) investigate the `oc` nuances (my least favorite option ;-) )

Reply:
I'll try b)

Reply:
ah yes, i understand the issue i think.  It does look like "openshift//templatename" with no `-n` argument should work.

Reply:
the reason is that "oc process" posts a "processtemplate" request object to whatever namespace you pass with "-n".  And regular users can't create that request object in the "openshift" namespace.  that's why the namespace//templatename syntax exists, so you say "get the template from namespace X, but process it in namespace Y"

Reply:
Interestingly  .... `oc adm policy who-can process templates -n openshift ` and `oc adm policy who-can get templates -n openshift ` return different results

Reply:
ah gotcha